onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -179,11 +182,10 @@ void BasicBackend::Infer(Ort::CustomOpApi& ort, OrtKernelContext* context) {
   std::lock_guard<std::mutex> lock(compute_lock_);
 
   size_t batch_size = 1;
-  // Get Input and Output tensors
-  auto input_tensors = GetInputTensors(ort, context, ie_cnn_network_, subgraph_context_.input_indexes);
```

Comment:
If we are not going to use GetInput Tensors, we should remove it

Reply:
> Done

Reply:
done